src/it/beta-just-testing/pom.xml

```
@@ -14,7 +14,7 @@
     <packaging>pom</packaging>
     <properties>
         <jenkins.version>2.7.3</jenkins.version>
-        <java.level>7</java.level>
+        <java.level>8</java.level>
```

```suggestion
        <java.level>7</java.level>
```

Please revert, should not impact the logic after the fix

---

The reason for this was that the default `animal-sniffer-annotations` version is updated in this PR to 1.18, and this fails when `java.level=7`:
```
[INFO] Restricted to JDK 1.7 yet org.codehaus.mojo:animal-sniffer-annotations:jar:1.18:provided contains org/codehaus/mojo/animal_sniffer/IgnoreJRERequirement.class targeted to JDK 8
[INFO] Ignoring requireUpperBoundDeps in com.google.guava:guava
[WARNING] Rule 2: org.apache.maven.plugins.enforcer.EnforceBytecodeVersion failed with message:
Found Banned Dependency: org.codehaus.mojo:animal-sniffer-annotations:jar:1.18
```

I can change it back to animal-sniffer-annotations:1.17, but is there a problem with this test change?

---

it's time to drop support for building against ancient cores.
java 8 has been available in Jenkins for a long time..

---

Phasing out Java 7 is already confirmed in https://groups.google.com/forum/#!msg/jenkinsci-dev/SdMOMKs7XIQ/_XADhodoBQAJ , no need in extra approvals for it.

If the Jenkins Version in this test is bumped to 2.60.3, everything will be fine IMHO